fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2357,6 +2357,7 @@ informationFunctionExpression
     | name = DATABASE '(' ')'
     | name = SCHEMA '(' ')'
     | name = USER '(' ')'
+    | name = SESSION_USER '(' ')'
```

Comment:
You use simpleFunctionCall rather than informationFunctionExpression to parse the `session_user()` function. So you can delelte this line. Besides, you use a implicit token in parser rule file which is not recommended.

Reply:
yes agree. i could have use simpleFunctionCall. but the other functions of similar type and behavior are implemented as informationFunctionExpression. so logically it felts right to put in along with similar functions.

Reply:
@packy92 have fixed the issue.